protocol/validators.re

```diff
@@ -3,7 +3,7 @@ open Crypto;
 
 // TODO: we should avoid dead validators to avoid double timeout, A(dead) -> B(dead) -> C
 [@deriving (yojson, ord)]
-type validator = {address: Address.t};
+type validator = {address: Wallet.t};
```

Comment:
I don't understand this change. Why call the field `address` if representing it by a Wallet? Seems like we now represent keys as wallets, and key_hashes as addresses, so isn't this inconsistent?

Reply:
Because on your PR this will change, from `Wallet.t` to `Adderss.t` back, so the field was kept to avoid noise on the review.

As soon as you apply your patch over this, it will look as `{ address: Address.t }`

Reply:
Awesome, thanks.